qutebrowser/completion/completionwidget.py

```diff
@@ -196,6 +196,47 @@ def _next_idx(self, upwards):
 
         raise utils.Unreachable
 
+    def _next_page(self, upwards):
+        """Essentially _next_idx with a for loop to increment by amount of elements per page (size of each page) instead of 1 each time
```

Comment:
wrap the line here, though I think a simpler summary would do.
Maybe "Return the index a page away from the selected index".
Mentioning implementation details like `with a for loop` shouldn't be necessary.